### ISSUE TITLE
Offer option to silence sclang startup messages

### ISSUE THREAD

**Issue by wasamasa** [supercollider/supercollider#3625]
I'm using `sclang` to run scripts. Is there any way to disable the startup messages displayed until the code is evaluated? Typically interpreters provide a flag or environment variable to make the program less chatty.

**Comment by brianlheim** [comment]
No, there is currently no way to do that, although I agree it would be nice.

**Comment by wasamasa** [comment]
Is there a way to work around this? Can I write a minimal client doing the same things as `sclang` and how would I do that? Is there documentation on writing custom clients?

**Comment by brianlheim** [comment]
You could wrap the call in a grep or other formatting program that filters out the unwanted messages. AFAIK there is no documentation on writing custom clients; you would start with the language client header in /include though: https://github.com/supercollider/supercollider/blob/develop/include/lang/SC_LanguageClient.h

**Comment by wasamasa** [comment]
The downside of using something like `grep` is that it alters user input going towards the client. I use a read from `/dev/stdin`, this breaks once I add `grep`, `sed` or `tail` to the mix.

**Comment by brianlheim** [comment]
Sorry, I don't understand, can you please show what you are doing?

**Comment by wasamasa** [comment]
You can find my current code on https://github.com/kanaka/mal/tree/supercollider-implementation/supercollider.

I wonder whether I could hand in a PR similar to https://github.com/supercollider/supercollider/pull/2431 but with all startup messages directed to stderr instead. That way people like me could just redirect them to `/dev/null` and the rest would still see them.
